src/components/EditTraceCommons.jsx

```diff
@@ -234,16 +235,15 @@ const TraceDatePicker = ({ onChange, value, disabled }) => {
   return (
     <Row gutter={16}>
       <Col className="gutter-row" span={10}>
-        <Form.Item label="Date" className="custom-form-item">
+        <Form.Item
+          label="Date"
+          className="custom-form-item"
+          rules={[{ required: true, message: 'Date is required' }]}
+        >
```

Comment:
![image](https://user-images.githubusercontent.com/5684607/127834315-a7507e37-42e3-4af0-a2bd-e14db06eea63.png)

I submit the trace without date and there is no error in form, and saving threw error in notification!

Reply:
We had this problem with other trace types (bounty, etc) because ant doesn't have required mark for datePicker. I solved it by adding default picker value for date.